chapter_multilayer-perceptrons/underfit-overfit.md

```diff
@@ -1,9 +1,9 @@
-```{.python .input}
+```{.python .input  n=1}
 %load_ext d2lbook.tab
 tab.interact_select(['mxnet', 'pytorch', 'tensorflow'])
 ```
 
-# Model Selection, Underfitting, and Overfitting
+# Model Selection
```

Comment:
Perhaps we can rename this file as "model-selection.md" and the other as "underfit-overfit.md", making it easier to locate the desired notebook when navigating through the "chapter_multilayer-perceptrons" directory.

Reply:
Indeed. The issue is that we're going to break some links but we should do this once properly.